tests/unit/test_graphstates.py

```diff
@@ -90,8 +90,15 @@ def test_index(cls, random_graph):
 class TestCVHelpers:
     """Tests for CVLayer helper functions."""
 
-    @classmethod
-    def test_SCZ_mat(cls, random_graph):
+    @pytest.mark.parametrize(
```

Comment:
Why decorator was removed here? In the following, we have "no-self-use" and there will be a pylint warning. I think we must revert to using @classmethod or @staticmethod here or some other solution.

Reply:
This is related to the discussion happening on the other PR [here](https://github.com/XanaduAI/flamingpy/pull/11#discussion_r847758935).

Reply:
Thanks all, as agreed I will remove non-pytest decorators from all test files and add `pylint: disable` as needed for now.